src/native-automation/request-pipeline/safe-api.ts

```
@@ -8,18 +8,25 @@ import ErrorReason = Protocol.Network.ErrorReason;
 import { isRequestPausedEvent } from '../utils/cdp';
 import { ContinueRequestArgs, SessionId } from '../types';
 
-const INVALID_INTERCEPTED_RESPONSE_ERROR_MSG = 'Invalid InterceptionId.';
+const INVALID_INTERCEPTED_RESPONSE_ERROR_MSG    = 'Invalid InterceptionId.';
+const SESSION_WITH_GIVEN_ID_NOT_FOUND_ERROR_MSG = 'Session with given id not found.';
```

I guess it's time to use array for the ignored error messages:
```js
const IGNORED_ERROR_MESSAGES = [
    'Invalid InterceptionId.',
    'Session with given id not found.'
];
```